lib/thrift/generator/server.ex

```diff
@@ -0,0 +1,143 @@
+defmodule Thrift.Generator.Server do
+  @moduledoc false
+
+  defmodule Framed do
+    @moduledoc false
+
+    alias Thrift.Generator.{
```

Comment:
I dunno, I've had it both ways and think that for > 3 aliases it saves enough to be worthwhile.